python/tvm/driver/tvmc/frontends.py

```diff
@@ -154,7 +154,8 @@ def load(self, path):
         # pylint: disable=C0415
         import onnx
 
-        model = onnx.load(path)
+        # pylint: disable=E1101
```

Comment:
Given this is repeated several times, do you think it's worth disabling for the whole file?

Reply:
Generally, being explicit and specific in lint exceptions are better, because this specific lint error (E1101) comes when accessing external imports which are not installed ci_lint.  If we disable it globally, we will miss the places where we want to capture it when we are accessing objects/modules defined in the code base.

Reply:
Yes, I'd like to keep in the smallest context as possible so we don't overkill the linter.